### ISSUE TITLE
Add vertical scroll bar to the SitRep Filter's drop-down menu

### ISSUE THREAD

**Issue by Oberlus** [freeorion/freeorion#3875]
Feature request
===============

In small screens, the list of sitrep types to block/unblock in the drop-down menu of the Filters button might be longer than the available screen space. The lack of a scroll bar makes impossible to select some of the sitrep types.
A handy workaround is to right-click in the sitreps and use the block options there. But unblocking using that system is less handy, as you cannot unblock a given type of sitrep but all of them at once.

Forum thread with screenshots: https://www.freeorion.org/forum/viewtopic.php?p=112036#p112036

**Comment by geoffthemedio** [comment]
might be easier to do sub menus like the objects list columns picker

**Comment by Vezzra** [comment]
> might be easier to do sub menus like the objects list columns picker

Like suggested in this recent feature request: #3852

**Comment by Oberlus** [comment]
And in that case a scroll bar seems unnecesary. Closing this one.

**Comment by Vezzra** [comment]
Superseded by better proposal in #3852
